src/lib/testing/constraint-system.ts

```diff
@@ -85,6 +86,37 @@ function constraintSystem<Input extends Tuple<CsVarSpec<any>>>(
   });
 }
 
+/**
+ * Convenience function to run {@link constraintSystem} on the method of a {@link ZkProgram}.
+ *
+ * @example
+ * ```ts
+ * const program = ZkProgram({ methods: { myMethod: ... }, ... });
+ *
+ * constraintSystem.fromZkProgram(program, 'myMethod', contains('Rot64'));
+ * ```
+ */
+constraintSystem.fromZkProgram = function fromZkProgram<
```

Comment:
This is a continuation of #1241. I figured I could make writing constraint system tests for zkprogram methods even easier by reading the input types off the zkprogram.